frost-core/src/benches.rs

```diff
@@ -0,0 +1,201 @@
+//! Ciphersuite-generic benchmark functions.
+
+use std::collections::HashMap;
+
+use criterion::{BenchmarkId, Criterion, Throughput};
+use rand_core::{CryptoRng, RngCore};
+
+use crate::{batch, frost, Ciphersuite, Signature, SigningKey, VerifyingKey};
+
+struct Item<C: Ciphersuite> {
+    vk: VerifyingKey<C>,
+    sig: Signature<C>,
+}
+
+fn sigs_with_distinct_keys<C: Ciphersuite, R: RngCore + CryptoRng + Clone>(
+    rng: &mut R,
+) -> impl Iterator<Item = Item<C>> {
+    let mut rng = rng.clone();
+    std::iter::repeat_with(move || {
+        let msg = b"Bench";
```

Comment:
###### Optional

Since the same message is used in two more places, it might be worth defining it as a function parameter or a constant outside of the fn.